transformers/avayaVoiceChannel/transformer.ts

```diff
@@ -0,0 +1,231 @@
+/** 
+ * Avaya voice channel
+ * version 0.1
+ */
+const COGNIGY_BASE_URL = 'https://endpoint-trial.cognigy.ai/';
+
+createRestTransformer({
+
+	/**
+	 * This transformer is executed when receiving a message
+	 * from the user, before executing the Flow.
+	 *
+	 * @param endpoint The configuration object for the used Endpoint.
+	 * @param request The Express request object with a JSON parsed body.
+	 * @param response The Express response object.
+	 *
+	 * @returns A valid userId, sessionId, as well as text and/or data,
+	 * which has been extracted from the request body.
+	 */
+	handleInput: async ({ endpoint, request, response }) => {
+
+		/**
+		 * Extract the userId, sessionId and text
+		 * from the request. Example:
+		 *
+		 * const { userId, sessionId, text, data } = request.body;
+		 *
+		 * Note that the format of the request body will be different for
+		 * every Endpoint, and the example above needs to be adjusted
+		 * accordingly.
+		 */
+		const { body } = request as CPaaSRequest;
+		console.log('body='.concat(JSON.stringify(body)));
```

Comment:
Please add a boolean DEBUG_MODE to the top configuration and only print out the console logs when the debug mode is on.

Reply:
made the changes per suggestion.